adapters/mqtt-vertx/pom.xml

```diff
@@ -110,8 +136,28 @@
                       </exec>
                     </cmd>
                     <assembly>
-                      <basedir>/opt/hono</basedir>
-                      <descriptorRef>artifact</descriptorRef>
+                      <!--<basedir>/opt/hono</basedir>
+                      <descriptorRef>artifact</descriptorRef>-->
+                      <mode>dir</mode>
+                      <basedir>/</basedir>
+                      <inline>
+                        <fileSets>
+                          <fileSet>
+                            <directory>${project.basedir}/target/classes/certs</directory>
+                            <outputDirectory>etc/hono/certs</outputDirectory>
+                            <includes>
+                              <include>trustStore.jks</include>
```

Comment:
Agree. The current docker compose file is using the jks file for this reason I used that. Can I go ahead and changing here and the docker compose file as well ?